ci/prow/check-commit-content

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+#FIXME: check if there is a prow github token that can be use to increase the api rate limit from 60 req/h to 5000 req/h
+
+pr=$(curl -H 'Accept: application/vnd.github+json' \
+    https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/pulls/${PULL_NUMBER})
+
+pr_title=$(echo ${pr} | jq '.title')
+# The github API returns \r\n for new lines (and \r\n\r\n for 2 new lines).
+# In Unix, those are interpreted as regulars chars and not as new line,
+# therefore, I had to remove them it order make it work.
+pr_body=$(echo ${pr} | jq '.body' | sed 's/\\r\\n\\r\\n/ /g')
+commit_subject=$(git log -1 --pretty=%s)
+commit_body=$(git log -1 --pretty=%b)
+
+
+diff=$(diff <(printf '%s' ${pr_title}) <(printf '%s' \"${commit_subject}\") | wc -l)
+if [[ ${diff} != 0 ]] ; then
+    echo "
+    Commit and PR are out of sync. Make sure the following are identical:
+        * PR title   <-->    commit subject
+    "
+    exit 1
+fi
+
+diff=$(diff <(printf '%s' ${pr_body}) <(printf '%s' \"${commit_body}\") | wc -l)
+if [[ ${diff} != 0 ]] ; then
+    echo "
+    Commit and PR are out of sync. Make sure the following are identical:
```

Comment:
what do you say about printing the diff in case of a failure?

Reply:
Done.

Reply:
I might be missing something, but where do you print the diff between PR body and commit message?